tests/test_utils.go

```diff
@@ -104,6 +104,15 @@ func IsCIRun() bool {
 	return os.Getenv("CI") == "true"
 }
 
+// SecurePath replaces all dangerous characters in the path.
+func SecurePath(path string) string {
+	badChars := []string{"\\", "?", "%", "*", ":", "|", `"`, "<", ">"}
```

Comment:
May be overkill, but looks like there is a sanitize package for similar purposes - https://godoc.org/gopkg.in/kennygrant/sanitize.v1